src/components/Swap/Status/StatusDialog.tsx

```diff
@@ -28,19 +29,35 @@ interface TransactionStatusProps {
 }
 
 function TransactionStatus({ tx, onClose }: TransactionStatusProps) {
+  const [showConfirmation, setShowConfirmation] = useState(true)
   const Icon = useMemo(() => {
+    if (showConfirmation) {
+      return LargeArrow
+    }
     return tx.receipt?.status ? LargeCheck : LargeSpinner
-  }, [tx.receipt?.status])
+  }, [showConfirmation, tx.receipt?.status])
+  useEffect(() => {
+    // We should show the confirmation for 1 second,
+    // which should start after the entrance animation is complete.
+    const handle = setTimeout(() => {
+      setShowConfirmation(false)
+    }, 1000 + DialogAnimationLengthMs)
```

Comment:
can you separate the `1000` into a constant like `ONE_SECOND_IN_MS`? it can go into a new `timing.ts` file or something of ur choice